src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -47,6 +48,15 @@ $side-nav-color-gray-lightest: #f8f8f8; // I could not find the color in our des
     margin: 10px 23px;
   }
 
+  .line-open {
+    background: $side-nav-color-dark;
+    display: block;
+    height: 0.5px;
+    border: 0;
```

Comment:
So I think a lot of these style properties (including the ones that I added originally, sorry!) could actually make use of the [design system's classes](https://design.va.gov/utilities/) instead, that way we can omit a lot of this custom CSS! 🙂

Reply:
I will have another iteration of changes and will try to adjust these as well